### ISSUE TITLE
[devtools] info:models cmd generates error `$cache is not configured`

### ISSUE THREAD

**Issue by Khairulfajeri** [phpMv/ubiquity#72]
![DeepinScreenshot_select-area_20190921175822](https://user-images.githubusercontent.com/18585608/65371878-e9f91400-dc9a-11e9-87fc-88de19d5302d.png)
can help me for this?

**Comment by jcheron** [comment]
Thanks for reporting @Khairulfajeri 
This is a devtool bug, which will be fixed in the next commit (and therefore in version 1.2.9 of devtools).

The upgrade to devtools 1.2.9 will require Ubiquity 2.3.1

While waiting for the next release of these 2 versions, you can switch to the master branches of each package.

As soon as a commit sets this issue, you can use this composer.json:
```json
{
    "require": {
        "php": "^7.1",
        "twig\/twig": "^2.0",
        "phpmv/ubiquity": "dev-master",
        "phpmv\/php-mv-ui": "dev-master"
    },
    "require-dev": {
        "monolog\/monolog": "^1.24",
        "mindplay\/annotations": "^1.3",
        "phpmv\/ubiquity-dev": "dev-master",
        "phpmv\/ubiquity-webtools": "dev-master",
        "czproject\/git-php": "^3.13"
    },
    "autoload": {
        "psr-4": {
            "": "app\/"
        }
    }
,
	"minimum-stability": "dev",
	"prefer-stable": true
}
```

**Comment by jcheron** [comment]
And [that](https://github.com/phpMv/ubiquity-devtools/commit/4ac56f134b761716bf83402c673a56ea6aa35ead) worked?

**Comment by Khairulfajeri** [comment]
Yeah, its solved

Pada tanggal Min, 22 Sep 2019 21.59, jcheron <notifications@github.com>
menulis:

**Comment by jcheron** [comment]
Thanks @Khairulfajeri !
Available in *devtools 1.2.9*
